src/sidebar/components/ThreadCard.js

```diff
@@ -63,13 +63,26 @@ function ThreadCard({ frameSync, thread }) {
   // parent component but the `Thread` itself has not changed.
   const threadContent = useMemo(() => <Thread thread={thread} />, [thread]);
 
+  // Handle requests to give this thread keyboard focus.
+  const focusRequest = store.annotationFocusRequest();
+  const cardRef = useRef(/** @type {HTMLElement|null} */ (null));
+  useEffect(() => {
+    if (focusRequest !== thread.id || !cardRef.current) {
+      return;
+    }
+    cardRef.current.focus();
+    store.clearAnnotationFocusRequest();
+  }, [focusRequest, store, thread.id]);
+
   return (
     /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
     <Card
       classes={classnames('p-3 cursor-pointer', {
         'is-focused': isFocused,
       })}
       data-testid="thread-card"
+      elementRef={cardRef}
+      tabIndex={0}
```

Comment:
Can you talk a little bit about the `tabIndex` strategy here?

Reply:
We need to assign a `tabIndex` to make the annotation focusable. We could either use `-1` to allow only programmatic focus, or `0` as here to include the card in the tab order. How this works really depends on how navigating the annotation list works. Currently `Thread` and `Annotation` components use `<section>` and `<article>` respectively so you can use article landmarks to navigate the visible ones. Using `-1` here would be the choice that doesn't impact other aspects of keyboard navigation.

Reply:
I have reset the `tabIndex` value to -1, so the keyboard tab sequence in the sidebar is the same as before. We might want to revisit how keyboard navigation of annotation cards works in future, but on reflection that's out of scope for this PR.